gradle.properties

```diff
@@ -1 +1 @@
-version=2.2.1-SNAPSHOT
+version=2.3.0-SNAPSHOT
```

Comment:
Don't change the version string. We use major.minor to reflect the version of gradle it is tested against. So this is 2.2.x.

I'll put out a 2.3.x using gradle 2.3 this week or next.